content/en/contribute/code/releasing.md

```diff
@@ -10,7 +10,34 @@ aliases: >
 
 ## CHT Core
 
-[Create an issue](https://github.com/medic/cht-core/issues/new/choose) for either a Major/Minor or Patch release issue template and assign it to the release manager to follow the process.
+A release is a set of changes that is planned to be used by at least one implementation that uses the CHT.
+
+### Building & Releasing CHT Core Changes
+
+The high-level steps for a release are as follows:
+
+* The team sees an opportunity they want to go after. The opportunity addresses a need of at least one implementation that uses the CHT and will be used by that implementation after the release.
```

Comment:
Who is "the team" here? Is it a Focused Working Group, or Medic's Product team? Given that this is the first use of "the team" on this page, worth being clearer about it.

Reply:
By team, I meant FWG. Just clarified it, thanks.